### ISSUE TITLE
Background and foreground colors do not have a sufficient contrast ratio on User Details page

### ISSUE THREAD

**Issue by megtera** [CMU-313/Mayan-EDMS#93]
There are two failing elements on the user details page related to the low-contrast text. The current accessibility score is 81.
